Runtime/Scripts/RGBotConfigs/RGEntity.cs

```diff
@@ -0,0 +1,56 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace RegressionGames.RGBotConfigs
+{
+    public class RGEntity : MonoBehaviour
+    {
+        [Header("General Information")] [Tooltip("Does this object represent a human/bot player ?")]
+        public bool isPlayer;
+
+        [Tooltip("A type name for associating like objects in the state")]
+        public string objectType;
+
+        // this is used in our toolkit to understand which things would need dynamic models
+        [Tooltip("Is this object spawned during runtime, or a fixed object in the scene?")]
+        public bool isRuntimeObject = false;
+
+        [Header("3D Positioning")] 
+        public bool syncPosition = true;
```

Comment:
@addisonbgross Note that the location of these fields to remove for REG-1303 is being moved to a new class here.